README.md

```diff
@@ -1,26 +1,119 @@
-<div align="center">
-    
-<h1>LetsBuildTogether Community (LBT) </h1>
+<img src="https://avatars.githubusercontent.com/u/102895308?s=200&v=4" width="50" alt="lbt_logo"/>
+<h1>Lets Build Together Community (LBT) </h1>
+
+`Join Our organization`
+
+<h3>LBT Community is a student-run organization on Discord. We provide resources, internship opportunities, and various tech programs that help students advance their careers. We also have student mentors who give their time to share their expertise with other student enthusiasts. In addition, we host events on various technologies.</h3>
+
+<h2> Join Us </h2>
+<p >
+    <a  href="https://discord.gg/TDzPbqwy"><img src="https://github.com/helper-uttam/Website/blob/main/assets/discord.png?raw=true"  width="50" alt="Join our Discord community here" /></a>
+  &nbsp; &nbsp; &nbsp;
+    <a  href="https://www.linkedin.com/company/letsbuildtogether/mycompany/"><img src="https://github.com/helper-uttam/Website/blob/main/assets/linkedin.png?raw=true" width="50" alt="LinkedIn" /></a>
+     &nbsp; &nbsp; &nbsp;
+        <a  href="https://twitter.com/LBT_Community"><img src="https://github.com/helper-uttam/Website/blob/main/assets/twitter_logo.png?raw=true" width="50" alt="Twitter" /></a>
+
+ If you would like to join our GitHub organisation, raise an issue on this [repo](https://github.com/Lets-Build-Together/Website) and Create PR.
+
+---
+
+
+
+
+#  Guide For Pull Requests
+
+Repo for you to raise a Pull Request for practice.
+
+**Just raise your issue if you want to upgrade other wise work on exsisting issues**
```

Comment:
this should be "Feel free to raise an issue if you wish to contribute and add values to our community, our maintainers will review your issue very soon"